src/www/components/landingpage/LandingPage.scss

```
@@ -0,0 +1,612 @@
+.container {
+  background-image: linear-gradient(  rgb(129, 129, 129) ,white);
+  height: 325vh; //parallax(110vh) + 2 pages(200vh) + 15vh more empty space
+  max-width: 100vw;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Times New Roman', Times, serif;
+
+  .thirdpage{
+    
+    .phrase{
+      margin-top: 20vh;
+      margin-bottom: 10vh;
+      float: left;
+      margin-left: 1  5vw;
+      font-size: calc(3.5vh + 1vw);
+    }
+
+    .parallaxbg{
+      
+      width: 74vw;
+      height: 65vh;
+      margin-left: 13vw;
+      align-items: center;
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      border-radius: 0.5rem;
+      
+      
+      .panel{
+        width: 80vw;
+        height: 65vh;   //same as parallaxbg height 
+        flex-wrap: wrap;
+        align-items: center;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        margin-left: 1rem;
+        margin-right: 1rem;
```

You can replace those two lines with:
```suggestion
        margin: 0 1rem;
```